apps/mobile/src/screens/PostEditor/PostEditor.styles.js

```diff
@@ -1,4 +1,5 @@
 import { StyleSheet } from 'react-native'
+import { twBackground } from 'style/colors'
```

Comment:
should have mentioned this yesterday but slight preference for something like `themeBackground` here, since its not about tailwind perse

Reply:
I probably agree with you there, but either way this is a one-time'r to get us through this release and not meant to be establishing (and very much would not want it to establish) any sort of new pattern if that makes sense. You did see the note in colors.js about how I got to this background color form the tailwind theme yes? Due to transparency values it actually had to be computed manually...